frontend/src/metabase/styled-components/theme/space.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns a pixel amount: 4px, 8px, 16px, on to 128px
+ * @param {number} level must be an integer between 0 and 5
+ * @returns {string}
+ */
+export function space(level = 0) {
+  const levels = [4, 8, 16, 32, 64, 128];
+  const spaceInteger = levels[level];
+
+  return spaceInteger ? spaceInteger + "px" : "";
+}
```

Comment:
I think we should define the `levels` array above the `space` function, so we don't spawn an array each time `space` is invoked

Reply:
Yup, changed, thank you.